jtiledownloader/README.md

```diff
@@ -0,0 +1,32 @@
+# jTileDownloader snap
+
+This project creates a working snap of
+[jTileDownloader](http://wiki.openstreetmap.org/wiki/JTileDownloader), a Java
+app to download OpenStreetMap tiles for offline map caching.
+
+Snapping inspired by [uNav's instructions for offline
+caching](https://unav-go.github.io/offline/).
+
+To get this done, we need to do the following:
+ - use the copy plugin to download and unpack a .zip file into a .jar file
+ - use a custom launcher to set up the environment variables required for
+   Java and use `java -tar` to execute the .jar file
+
+## Current state
+
+Working features:
+ - Tested in devmode only
+ - App GUI starts
+ - Tiles download requests seem to be blocked, but this seems to be an issue
+   with the app running outside the snap too
+
+Known issues:
+ - Theming does not work.
```

Comment:
If the Java Swing app uses the GTK LaF then making it work is the same as with normal GTK apps:
https://github.com/filebot/filebot/blob/master/installer/snappy/filebot.sh#L15